vscode-udfs/README.md

```diff
@@ -0,0 +1,93 @@
+<!-- title: How to create UDFs with Confluent for VS Code -->
```

Comment:
I think that the title should mention CC for AF, and also make it clear that it's the sample template. i.e., this isn't explaining to users how to develop a UDF though arguably it should get into that more (comment on that below). I would suggest `How to use Confluent for VS Code to create a sample UDF in Confluent Cloud for Apache Flink` or `How to create a sample UDF in Confluent Cloud for Apache Flink using Confluent for VS Code`